workspaces/announcements/plugins/announcements-backend/CHANGELOG.md

```diff
@@ -0,0 +1,256 @@
+# @backstage-community/plugin-announcements-backend
```

Comment:
If we are going to keep the old changelogs then they should reflect the history. Looking at this you would think this plugin was alway part of the `@backstage-community` scope.